Interfaces/DBView.xib

```diff
@@ -3453,6 +3453,33 @@ Gw
                                         <color key="backgroundColor" name="controlColor" catalog="System" colorSpace="catalog"/>
                                     </textFieldCell>
                                 </textField>
+                                <popUpButton verticalHuggingPriority="750" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="wQQ-It-oAb">
+                                    <rect key="frame" x="115" y="68" width="171" height="22"/>
+                                    <autoresizingMask key="autoresizingMask" flexibleMaxX="YES" flexibleMinY="YES"/>
+                                    <popUpButtonCell key="cell" type="push" title="Item 1" bezelStyle="rounded" alignment="left" controlSize="small" lineBreakMode="truncatingTail" state="on" borderStyle="borderAndBezel" imageScaling="proportionallyDown" inset="2" selectedItem="LCH-j8-1Mj" id="d5b-XR-bub">
+                                        <behavior key="behavior" lightByBackground="YES" lightByGray="YES"/>
+                                        <font key="font" metaFont="message" size="11"/>
+                                        <menu key="menu" title="OtherViews" id="OOS-4Z-1Qg">
+                                            <items>
+                                                <menuItem title="Item 1" state="on" id="LCH-j8-1Mj"/>
+                                                <menuItem title="Item 2" id="InD-41-0xM"/>
+                                                <menuItem title="Item 3" id="iqD-tE-mQl"/>
```

Comment:
Nitpick - should we get rid of the dummy options?

Reply:
`Item 1, 2, 3` are also present for the others pop up (Table, column), I remove them too.